app/Http/Controllers/RequisitionController.php

```diff
@@ -52,26 +52,28 @@ public function filterAndExport(Request $request)
         $requisitions = $query->get();
 
         $exportData = $requisitions->map(function($requisition) use ($request) {
+            $department_date = $requisition->getRelation('events')->filter(function($item) {
+                return $item->type == 'Enviado para análise do departamento';
+            })->last();
+
+            $registered_date = $requisition->getRelation('events')->filter(function($item) {
+                return $item->type == 'Aguardando avaliação da CG';
+            })->last();
```

Comment:
Os nomes tão um pouco incoerentes com o o dado, já que o que está armazenado na variável é um objeto requisition e não uma data. Uma opção é fazer direto

```suggestion
            $department_date = $requisition->getRelation('events')->filter(function($item) {
                return $item->type == 'Enviado para análise do departamento';
            })->last()->created_at->format('d-m-Y');

            $registered_date = $requisition->getRelation('events')->filter(function($item) {
                return $item->type == 'Aguardando avaliação da CG';
            })->last()->created_at->format('d-m-Y');
```

Reply:
A primeira tentativa era direto, mas acabou tendo o mesmo problema do 'reviews', onde o requerimento procurado pode não existir, então '->created_at' pode dar erro. Vou arrumar os nomes.